### PR TITLE
Remove KonnectivityService feature gate & make Konnectivity generally available

### PR DESCRIPTION
**Pull request by rastislavs**
Signed-off-by: Rastislav Szabo <rastislav@kubermatic.com>

**What this PR does / why we need it**:
This PR makes Konnectivity GA by removing `KonnectivityService` feature gate and making it always available.

**Which issue(s) this PR fixes**:

Fixes #

**What type of PR is this?**
/kind feature

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change? Then add your Release Note here**:

```release-note
Remove KonnectivityService feature gate & make Konnectivity generally available
```

**Documentation**:

```documentation
https://github.com/kubermatic/docs/pull/1293
```

### CODE REVIEW

File: pkg/controller/seed-controller-manager/monitoring/resources.go [hunk 1]
```diff
@@ -49,7 +49,7 @@ func (r *Reconciler) getClusterTemplateData(ctx context.Context, client ctrlrunt
 	}
 
 	// Konnectivity is enabled if the feature gate is enabled and the cluster flag is enabled as well
```
Comment (mate4st) [inline pkg/controller/seed-controller-manager/monitoring/resources.go]: NIT: Since you removed the feature gate, this comment is invalid :)

Comment (rastislavs) [reply pkg/controller/seed-controller-manager/monitoring/resources.go]: thx, fixed ;)

**Pull request merged**